htdocs/admin/delais.php

```diff
@@ -343,16 +348,19 @@
 		print '<div>';
 		print '<div class="inline-block" style="padding-right: 20px">';
 		print img_weather($text, 0, $options);
-		print ' &lt;= <input type="text" size="2" name="MAIN_METEO_PERCENTAGE_LEVEL0" value="'.$conf->global->MAIN_METEO_PERCENTAGE_LEVEL0.'"/>&nbsp;%</td>';
+		print ' &lt;= <input type="text" size="2" name="MAIN_METEO_PERCENTAGE_LEVEL0" value="'.(isset($conf->global->MAIN_METEO_PERCENTAGE_LEVEL0) ? $conf->global->MAIN_METEO_PERCENTAGE_LEVEL0 : '0').'"/>&nbsp;%</td>';
 		print '</div><div class="inline-block" style="padding-right: 20px">';
 		print img_weather($text, 1, $options);
-		print ' &lt;= <input type="text" size="2" name="MAIN_METEO_PERCENTAGE_LEVEL1" value="'.$conf->global->MAIN_METEO_PERCENTAGE_LEVEL1.'"/>&nbsp;%</td>';
+		print ' &lt;= <input type="text" size="2" name="MAIN_METEO_PERCENTAGE_LEVEL1" value="'.(isset($conf->global->MAIN_METEO_PERCENTAGE_LEVEL1) ? $conf->global->MAIN_METEO_PERCENTAGE_LEVEL1 : '10').'"/>&nbsp;%</td>';
 		print '</div><div class="inline-block" style="padding-right: 20px">';
 		print img_weather($text, 2, $options);
-		print ' &lt;= <input type="text" size="2" name="MAIN_METEO_PERCENTAGE_LEVEL2" value="'.$conf->global->MAIN_METEO_PERCENTAGE_LEVEL2.'"/>&nbsp;%</td>';
+		print ' &lt;= <input type="text" size="2" name="MAIN_METEO_PERCENTAGE_LEVEL2" value="'.(isset($conf->global->MAIN_METEO_PERCENTAGE_LEVEL2) ? $conf->global->MAIN_METEO_PERCENTAGE_LEVEL2 : '20').'"/>&nbsp;%</td>';
 		print '</div><div class="inline-block" style="padding-right: 20px">';
 		print img_weather($text, 3, $options);
-		print ' &lt;= <input type="text" size="2" name="MAIN_METEO_PERCENTAGE_LEVEL3" value="'.$conf->global->MAIN_METEO_PERCENTAGE_LEVEL3.'"/>&nbsp;%</td>';
+		print ' &lt;= <input type="text" size="2" name="MAIN_METEO_PERCENTAGE_LEVEL3" value="'.(isset($conf->global->MAIN_METEO_PERCENTAGE_LEVEL3) ? $conf->global->MAIN_METEO_PERCENTAGE_LEVEL3 : '30').'"/>&nbsp;%</td>';
+		print '</div><div class="inline-block" style="padding-right: 20px">';
+		print img_weather($text, 4, $options);
+		print ' &lt;= <input type="text" size="2" name="MAIN_METEO_PERCENTAGE_LEVEL4" value="'.(isset($conf->global->MAIN_METEO_PERCENTAGE_LEVEL4) ? $conf->global->MAIN_METEO_PERCENTAGE_LEVEL4 : '40').'"/>&nbsp;%</td>';
```

Comment:
There is no level4 threshold value. Level 4 is all value that are higher than level3 max amount.

Reply:
@eldy i revert my commit for this file